mail-server/dovecot.nix

```diff
@@ -41,7 +41,7 @@ in
       protocols = [ "sieve" ];
 
       sieveScripts = {
-        before = builtins.toFile "spam.sieve" ''
+        after = builtins.toFile "spam.sieve" ''
```

Comment:
Can you elaborate on what the interaction of this and the line

```
sieve_default = file:/var/sieve/%u/default.sieve
sieve_default_name = default
```
is?

Reply:
The `sieve_default` script is executed first. If a matching rule is found in this script, the corresponding action is applied and `spam.sieve` is not executed (unless the action is `keep`). If no role matches, the Sieve interpreter continues to the `spam.sieve` script.